### PR TITLE
[doc][yba] Prerequisites reorganization

### PR DESCRIPTION
**Pull request by ddhodge**
Prerequisites reorganization

@netlify /preview/yugabyte-platform/yba-overview/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/prepare/_index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+headcontent: Install YugabyteDB Anywhere to begin creating and managing deployments.
+menu:
+  preview_yugabyte-platform:
+    parent: yugabytedb-anywhere
+    identifier: prepare
+weight: 605
+type: indexpage
+---
+
+YugabyteDB Anywhere is a control plane used to deploy and manage YugabyteDB database clusters. In addition to needing a server to host YugabyteDB Anywhere, you also need the necessary permissions to allow YugabyteDB Anywhere to deploy and provision the nodes that go to make up the dtabase clusters.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/prepare/_index.md:16]:
```suggestion
YugabyteDB Anywhere is a control plane used to deploy and manage YugabyteDB database clusters. In addition to needing a server to host YugabyteDB Anywhere, you also need the necessary permissions to allow YugabyteDB Anywhere to deploy and provision the nodes that go to make up the database clusters.
```

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+At a high level, you will perform the following tasks to install and use YugabyteDB Anywhere to manage database deployments:
+
+1. Install the management software (that is, YugabyteDB Anywhere).
+1. Create a Cloud Provider using YugabyteDB Anywhere.
+1. Create a database cluster (also referred to as a Universe) using YugabyteDB Anywhere.
+
+    The universe configuration and deployment uses the details from the cloud provider. If using Kubernetes, using Helm charts or the Kubernetes operator is also possible.
+
+### Pre-requisites
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/yba-overview.md:73]:
```suggestion
### Prerequisites
```

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+1. Rotation of CA and server certificates.
+1. Rotation of KMS keys used for database encryption at rest.
+1. Monitoring and alerting of database health.
+1. Monitoring performance and metrics, including finding slow queries and getting insights on performance tuning using Performance Advisor.
+
+YBA operations can be performed through a GUI, or via automation tools that use its exposed REST APIs or Terraform provider.
+
+## Architecture
+
+YugabyteDB Anywhere (YBA) runs on standalone VMs or K8s pods. Separately, YBA supports deployment of YugabyteDB database clusters on either VMs or K8s pods.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/yba-overview.md:51]:
```suggestion
YugabyteDB Anywhere (YBA) runs on standalone VMs or Kubernetes pods. Separately, YBA supports deployment of YugabyteDB database clusters on either VMs or Kubernetes pods.
```

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+type: docs
+---
+
+YugabyteDB Anywhere (YBA) is a self-managed database-as-a-service that allows you to deploy and operate YugabyteDB database clusters (also known as universes) at scale.
+
+## Features
+
+### Deployment
+
+You can use YBA to deploy YugabyteDB database clusters in a variety of configurations and environments. YugabyteDB's flexible fault tolerance options include single availability zone, multiple availability zone, multi region, and multi cloud. Additionally, you can optionally set up a single read replica cluster attached to a primary cluster, and set up disaster recovery across two YugabyteDB clusters.
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/yba-overview.md:21]: There are some other words like Performance Advisor, automation tools, yb-master, yb-tserver, node agent, Cloud provider( link to #what-is-a-cloud-provider) for which we should add links as they are YB-specific terms.

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+### Pre-requisites
+
+- Server hardware and software
+
+  - For YugabyteDB Anywhere - 1 or 2 servers (or pods if deployed on K8s) and Linux root permissions
+  - For database clusters (also called a universe) - 1, 3, 5, or more servers (or pods if deployed on Kubernetes) and Linux root permissions
+
+- Networking connectivity
+
+  - Between YBA and all of the DB nodes
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/yba-overview.md:82]: Between YBA and all nodes of the database cluster. Right?

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+  - Between all the DB nodes
+  - Optional network connectivity to the Internet for Linux OS repositories
+  - Additional networking connectivity for authenticating via an external Identity Provider, if backing up externally, or for other special purposes
+
+- Cloud permissions. These are required when using Kubernetes or a public cloud provider (AWS, GCP, or Azure). Not required when using an on-premises provider.
+
+### What is a Cloud Provider?
+
+A Cloud Provider (called an Infrastructure Provider Configuration in the YBA UI) describes your cloud. For example, the number of regions and zones. In the case of public IaaS clouds, what service account should YBA use to create servers/VMs in your cloud? You must create a cloud provider before you can create your first universe.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/yba-overview.md:91]:
```suggestion
A Cloud Provider (called an Infrastructure Provider Configuration in the YBA UI) describes your cloud. For example, the number of regions and zones. In the case of public IaaS clouds, what service account should YBA use to create servers/VMs in your cloud? You must create a cloud provider before you can create your first database cluster (universe).
```

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+1. Kubernetes (for example, VMware Tanzu, Red Hat OpenShift, or Managed Kubernetes Service)
+1. Public Cloud (AWS, GCP, or Azure)
+1. On-premises
+
+#### Kubernetes
+
+If you are deploying a universe to Kubernetes, use a Kubernetes provider.
+
+Note that if you want to deploy a database cluster to Kubernetes, YBA must itself also be deployed on K8s.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/yba-overview.md:103]:
```suggestion
Note that if you want to deploy a database cluster to Kubernetes, YBA must itself also be deployed on Kubernetes.
```

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+#### On-premises
+
+For all other cases, use an on-premises provider. Unlike the other two options, where YBA has privileges and creates the VMs (or pods) automatically, with an on-premises provider *you* create the server VMs manually.
+
+Use this option for any of the following situations:
+
+- To run your database cluster nodes truly on-premises (for example, on VMware or Nutanix).
+- To run your database cluster nodes on the public cloud, but, due to security policies or other restrictions, you cannot provide YBA with cloud permissions, or SSH access to cloud VMs. In this case, you must create your VMs and/or deploy your Linux OS manually.
+- To run a single [stretched cluster across multiple clouds](../../develop/multi-cloud/multicloud-setup/) (for example, one cluster with some nodes on AWS, others on GCP, and/or others on Azure).
+- Any other cases when you must have control over creating the VMs and/or Linux OS… and cannot give such control to YBA
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/yba-overview.md:125]:
```suggestion
- Any other cases where you may have to control creating the VMs and/or Linux OS, and cannot provide access to YBA.
```

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+
+Read all the pre-requisites first, before proceeding.
+
+Fill out the pre-requisites form as you go along. Start installing only after the pre-requisites form is completed.
+
+If you have development, staging, and production environments (with multiple clusters (universes) per environment), deploy one YBA instance (or active/passive high availability pair) for each environment.
+
+## Limitations
+
+YBA can be deployed to on-premises VMs, public cloud VMs, or K8s. The same is true for universes. While these are usually de-coupled and independent deployment decisions, there is one case where a dependency exists: if you want to deploy a database cluster to Kubernetes, YBA must also be deployed on K8s.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/yba-overview.md:139]:
```suggestion
YBA can be deployed to on-premises VMs, public cloud VMs, or Kubernetes. The same is true for universes. While these are usually de-coupled and independent deployment decisions, there is one case where a dependency exists, that is, if you want to deploy a database cluster to Kubernetes, YBA must also be deployed on Kubernetes.
```

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+- To run your database cluster nodes truly on-premises (for example, on VMware or Nutanix).
+- To run your database cluster nodes on the public cloud, but, due to security policies or other restrictions, you cannot provide YBA with cloud permissions, or SSH access to cloud VMs. In this case, you must create your VMs and/or deploy your Linux OS manually.
+- To run a single [stretched cluster across multiple clouds](../../develop/multi-cloud/multicloud-setup/) (for example, one cluster with some nodes on AWS, others on GCP, and/or others on Azure).
+- Any other cases when you must have control over creating the VMs and/or Linux OS… and cannot give such control to YBA
+
+With the on-premises provider, after creating VMs manually (that is, outside of YBA), you will add them to your on premises provider's free pool of servers. Subsequently, when creating the universe, database nodes are taken from the on-premises provider's free pool of servers and added to the universe.
+
+## Best Practices
+
+Read all the pre-requisites first, before proceeding.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/yba-overview.md:131]:
```suggestion
Make sure you read the [Prerequisites](#prerequisites) before proceeding.
```

**Review by aishwarya24 — COMMENTED** [review]
some comments for Overview.

File: docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes.md [hunk 11]
```diff
@@ -0,0 +1,351 @@
...
+
+![Marketplace success](/images/ee/aws-setup/marketplace-success-almalinux.png)
+
+  </div>
+
+  <div id="gcp" class="tab-pane fade" role="tabpanel" aria-labelledby="gcp-tab">
+
+## GCP
+
+The following permission is required on the GCP project where you will deploy:
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes.md:181]:
```suggestion
The following permission is required on the GCP project where you will deploy.
```

File: docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes.md [hunk 12]
```diff
@@ -0,0 +1,351 @@
...
+
+- [Register an application](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) in the Azure portal so the Microsoft identity platform can provide authentication and authorization services for your application. Registering your application establishes a trust relationship between your application and the Microsoft identity platform.
+
+- [Assign a managed identity](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/qs-configure-portal-windows-vm) to the Azure VM hosting YugabyteDB Anywhere.
+
+For information on assigning roles to applications, see [Assign a role to an application](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#assign-a-role-to-the-application); and assigning roles for managed identities, see [Assign Azure roles using the Azure portal](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=delegate-condition) in the Microsoft Azure documentation.
+
+If are registering an application, record the following information about your service account. You will need to provide this information later to YBA.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes.md:216]:
```suggestion
If you are registering an application, record the following information about your service account. You will need to provide this information later to YBA.
```

File: docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes.md [hunk 13]
```diff
@@ -0,0 +1,351 @@
...
+As a prerequisite, you need a service account in the Kubernetes cluster and to generate a corresponding KUBECONFIG file.
+
+Perform the following:
+
+- [Create a `yugabyte-platform-universe-management` service account](#create-a-service-account).
+- [Create a `kubeconfig` file](#create-a-kubeconfig-file) of the service account you created to configure access to the Kubernetes cluster.
+
+Do this for each Kubernetes cluster if you are doing a multi-cluster setup.
+
+If YBA is itself installed on Kubernetes, you can use its existing service account to discover details about the Kubernetes cluster and auto-fill the provider configuration. You can then modify these settings to further customize the provider. See [Create a provider](../../../configure-yugabyte-platform/kubernetes/).
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes.md:241]: I thought there is no "If YBA is itself installed on Kubernetes" - YBA must be installed on Kubernetes to deploy the YBA universe nodes as well. Right?

**Review by aishwarya24 — COMMENTED** [review]
some comments. Will revisit shortly

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-kubernetes.md [hunk 14]
```diff
@@ -0,0 +1,220 @@
...
+
+If the value of `core_pattern` contains a `|` pipe symbol (for example, `|/usr/share/apport/apport -p%p -s%s -c%c -d%d -P%P -u%u -g%g -- %E` ), the core dump is being redirected to a specific collector on the underlying Kubernetes node, with the location depending on the exact collector. To be able to retrieve core dump files in case of a crash within the Kubernetes pod, it is important that you understand where these files are written.
+
+If the value of `core_pattern` is a literal path of the form `/var/tmp/core.%p`, no action is required on your part, as core dumps will be copied by the YugabyteDB node to the persistent volume directory `/mnt/disk0/cores` for future analysis.
+
+Note the following:
+
+- ulimits and sysctl are inherited from Kubernetes nodes and cannot be changed for an individual pod.
+- New Kubernetes nodes might be using [systemd-coredump](https://www.freedesktop.org/software/systemd/man/systemd-coredump.html) to manage core dumps on the node.
+
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-kubernetes.md:54]: - Is this step not required - https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes/#configure-storage-class. Compared it to the current preview docs page

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 15]
```diff
@@ -9,39 +9,27 @@ aliases:
...
   </li>
 
-  <li>
-    <a href="../openshift/" class="nav-link">
-      <i class="fa-brands fa-redhat"></i>OpenShift</a>
-  </li>
-
 </ul>
 
 Use YBA Installer to install YBA on a host, either online or airgapped. YBA Installer performs preflight checks to validate the workspace is ready to run YBA.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:35]: YBA Installer performs preflight checks to validate if the workspace is ready to run YBA.

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-kubernetes.md (hunk 14 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-kubernetes.md:54]: Was put under hardware

File: docs/content/preview/yugabyte-platform/prepare/networking.md [hunk 16]
```diff
@@ -0,0 +1,127 @@
...
+    parent: prepare
+    weight: 10
+type: docs
+---
+
+YugabyteDB Anywhere needs to be able to access nodes that will be used to create universes, and the nodes that make up universes need to be accessible to each other and to applications.
+
+## Global port requirements
+
+![YugabyteDB Anywhere network and ports](/images/yb-platform/prepare/yba-networking.png)
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/prepare/networking.md:19]: Deliberate

File: docs/content/preview/yugabyte-platform/yba-overview.md [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+type: docs
+---
+
+YugabyteDB Anywhere (YBA) is a self-managed database-as-a-service that allows you to deploy and operate YugabyteDB database clusters (also known as universes) at scale.
+
+## Features
+
+### Deployment
+
+You can use YBA to deploy YugabyteDB database clusters in a variety of configurations and environments. YugabyteDB's flexible fault tolerance options include single availability zone, multiple availability zone, multi-region, and multi-cloud. Additionally, you can optionally set up a single [read replica](../../architecture/docdb-replication/read-replicas/) cluster attached to a primary cluster, and set up disaster recovery across two YugabyteDB clusters.
```
Comment (yushenng) [inline docs/content/preview/yugabyte-platform/yba-overview.md:21]: Reword this to:

You can use YBA to deploy YugabyteDB database clusters in a variety of configurations and environments. YugabyteDB's flexible deployment options allow clusters to be deployed to handle a wide variety of fault tolerance conditions, including: single node failures, multiple node failures, single availability zone failures, multiple availability zone failures, single region failures, single cloud provider failures, and more. Although we use the term “database cluster” here (and throughout the docs) for clarity, the YBA product technically and precisely calls these “universes”.   A universe always contains one and only one primary cluster.  Additionally, within a universe, you can optionally set up a single [read replica](https://deploy-preview-21876--infallible-bardeen-164bc9.netlify.app/preview/architecture/docdb-replication/read-replicas/) cluster attached to a primary cluster.   And, you can set up disaster recovery across two YugabyteDB universes.

File: docs/content/preview/yugabyte-platform/prepare/networking.md [hunk 18]
```diff
@@ -0,0 +1,127 @@
...
+
+  </div>
+
+  <div id="gcp" class="tab-pane fade" role="tabpanel" aria-labelledby="gcp-tab">
+
+In GCP, YBA deploys all database cluster VMs in a single [GCP VPC](https://cloud.google.com/vpc/docs/vpc). You need to identify the VPC and specific subnets for the regions in which VMs are deployed and configure the appropriate [GCP firewall rules](https://cloud.google.com/firewall/docs/using-firewalls) to open the required ports. When the YBA VM and the DB VMs are deployed in separate GCP VPCs, these VPCs need to be peered to enable connectivity. Typically, routes for the peered VPCs are [set up automatically](https://cloud.google.com/vpc/docs/vpc-peering#route-exchange-options) but may need to be configured in some cases.
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/networking.md:106]:
```suggestion
In GCP, YBA deploys all database cluster VMs in a single [GCP VPC](https://cloud.google.com/vpc/docs/vpc). You need to identify the VPC and specific subnets for the regions in which VMs are deployed and configure the appropriate [GCP firewall rules](https://cloud.google.com/firewall/docs/using-firewalls) to open the required ports. When the YBA VM and the DB VMs are deployed in separate GCP VPCs, these VPCs need to be peered to enable connectivity. Typically, routes for the peered VPCs are [set up automatically](https://cloud.google.com/vpc/docs/vpc-peering#route-exchange-options) but may need to be configured explicitly in some cases.
```

File: docs/content/preview/yugabyte-platform/prepare/networking-kubernetes.md [hunk 19]
```diff
@@ -0,0 +1,75 @@
...
+description: Added requirements for deploying YugabyteDB Anywhere on Kubernetes.
+headContent: Added requirements for YugabyteDB Anywhere on Kubernetes
+menu:
+  preview_yugabyte-platform:
+    identifier: networking-kubernetes
+    parent: networking
+    weight: 10
+type: docs
+---
+
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/networking-kubernetes.md:14]: Should we add a heading like "Basic requirements" here? Otherwise, it seems like the focus of this page is multi-cluster, which is not usually the common focus.

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md [hunk 20]
```diff
@@ -0,0 +1,189 @@
...
+  </div>
+
+  <div id="aws" class="tab-pane fade" role="tabpanel" aria-labelledby="aws-tab">
+
+Refer to [Hardware requirements](../../../deploy/checklist/#hardware-requirements) for database cluster node hardware requirements. In particular, note the following sections:
+
+- [CPU and RAM](../../../deploy/checklist/#cpu-and-ram)
+- [Verify support for SSE2 and SSE4.2](../../../deploy/checklist/#verify-support-for-sse2-and-sse4-2)
+- [Disks](../../../deploy/checklist/#disks)
+- [Amazon Web Services](../../../deploy/checklist/#amazon-web-services-aws)
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md:73]:
```suggestion
- [Amazon Web Services instance type and volume recommendations](../../../deploy/checklist/#amazon-web-services-aws)
```

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md [hunk 21]
```diff
@@ -0,0 +1,189 @@
...
+  </div>
+
+  <div id="gcp" class="tab-pane fade" role="tabpanel" aria-labelledby="gcp-tab">
+
+Refer to [Hardware requirements](../../../deploy/checklist/#hardware-requirements) for database cluster node hardware requirements. In particular, note the following sections:
+
+- [CPU and RAM](../../../deploy/checklist/#cpu-and-ram)
+- [Verify support for SSE2 and SSE4.2](../../../deploy/checklist/#verify-support-for-sse2-and-sse4-2)
+- [Disks](../../../deploy/checklist/#disks)
+- [Google Cloud](../../../deploy/checklist/#google-cloud)
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md:86]:
```suggestion
- [Google Cloud instance type and volume type recommendations](../../../deploy/checklist/#google-cloud)
```

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md [hunk 22]
```diff
@@ -0,0 +1,189 @@
...
+
+Using local disks comes with certain restrictions on Day 2 YBA operations, so we typically recommend using remote disks instead.
+
+## GCP instance templates
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md:90]: This doesn't seem to belong in the hardware requirements section because we don't require anything here?

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md [hunk 23]
```diff
@@ -0,0 +1,189 @@
...
+  </div>
+
+  <div id="azure" class="tab-pane fade" role="tabpanel" aria-labelledby="azure-tab">
+
+Refer to [Hardware requirements](../../../deploy/checklist/#hardware-requirements) for database cluster node hardware requirements. In particular, note the following sections:
+
+- [CPU and RAM](../../../deploy/checklist/#cpu-and-ram)
+- [Verify support for SSE2 and SSE4.2](../../../deploy/checklist/#verify-support-for-sse2-and-sse4-2)
+- [Disks](../../../deploy/checklist/#disks)
+- [Azure](../../../deploy/checklist/#azure)
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md:134]:
```suggestion
- [Azure instance type and disk type recommendations](../../../deploy/checklist/#azure)
```

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md [hunk 24]
```diff
@@ -0,0 +1,189 @@
...
+  name: yb-storage
+provisioner: kubernetes.io/gce-pd
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+parameters:
+  type: pd-ssd
+  fstype: xfs
+```
+
+The storage class for most cloud providers allows further configuration like IOPS and throughput and volume types. It is recommended to configure these parameters to match the recommended parameters as listed in [Public clouds](../../../deploy/checklist/#public-clouds). For more information on these configuration parameters, please refer to the [AWS](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/parameters.md), [GCP](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/ssd-pd), and [Azure](https://learn.microsoft.com/en-us/azure/aks/azure-disk-csi#create-a-custom-storage-class) documentation.
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-hardware.md:186]: Maybe this should be a separate bullet and be placed before the example so that the example kind of summarizes all the recommendations above.

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+
+Depending on the cloud provider type and permissions you grant, you may have to install all of these requirements manually, or YugabyteDB Anywhere (YBA) will install it all automatically.
+
+##### Linux OS
+
+YBA supports deploying YugabyteDB on a variety of [operating systems](../../../reference/configuration/operating-systems/).
+
+AlmaLinux OS 8 disk images are used by default, but you can specify a custom disk image and OS.
+
+Linux must be configured with a `yugabyte` user, sufficient file descriptor limits (that is, ulimits), and more. With the exception of a fully manual on-premises provisioning, this configuration is performed automatically by YBA.
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md:28]: Given this line applies only to fully manual and it is covered in the prep section there, should we remove it to avoid confusion for all other cases?

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md [hunk 26]
```diff
@@ -0,0 +1,49 @@
...
+
+##### Additional software
+
+YBA requires the following additional software to be pre-installed on nodes:
+
+- Python 3.6-3.8
+- OpenSSH Server. Allowing SSH is recommended but optional. It can be skipped in some on-premises deployment approaches; all other workflows require it.
+- Tar
+- unzip
+- policycoreutils-python
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md:38]: Minor: lower case would match the typical package names better for the last 3 lines

For OpenSSH Server, capital letters is ok because we are not specifying a package name.

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-kubernetes.md [hunk 27]
```diff
@@ -0,0 +1,220 @@
+---
+title: YugabyteDB Anywhere Kubernetes software requirements
+headerTitle: Software requirements for Kubernetes
+linkTitle: Kubernetes
+description: Software requirements for Kubernetes.
+headContent: Prepare a VM for deployment in a universe
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-kubernetes.md:6]:
```suggestion
headContent: Prepare a Kubernetes cluster for YugabyteDB deployments
```

**Comment by iSignal** [comment]
https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes/#enable-tectia-ssh probably needs a new home

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md [hunk 28]
```diff
@@ -0,0 +1,47 @@
...
+- tar
+- unzip
+- policycoreutils-python
+
+##### Additional software for airgapped deployment
+
+Additionally, if not connected to the public Internet (that is, airgapped); and not connected to a local Yum repository that contains the [additional software](#additional-software), database cluster nodes must also have the following additional software pre-installed:
+
+- libselinux-python (yum package) for python 3.6, [selinux](https://pypi.org/project/selinux/) (PyPI package) for certain higher versions of python (for various Ansible tasks to work under the hood)
+- libcgroup and libcgrouptools (for planned future use of the cgconfig service, for cgroups; can be omitted for YBA versions earlier than v2024.1)
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md:43]:
```suggestion
- libcgroup and libcgroup-tools (for planned future use of the cgconfig service, for cgroups; can be omitted for YBA versions earlier than v2024.1)
```

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md [hunk 29]
```diff
@@ -0,0 +1,47 @@
...
+- rsync, openssl (all linux)
+- glibc-locale-source, glibc-langpack-en
+- libatomic, libncurses6 (for aarch64)
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md:46]:
```suggestion
- libatomic (for Redhat-based aarch64)
- libatomic1, libncurses6 (for Debian-based aarch64)
```

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md [hunk 30]
```diff
@@ -0,0 +1,47 @@
...
+- OpenSSH Server. Allowing SSH is recommended but optional. Using SSH can be skipped in some on-premises deployment approaches; all other workflows require it. [Tectia SSH](../../create-deployments/connect-to-universe/#enable-tectia-ssh) is also supported.
+- tar
+- unzip
+- policycoreutils-python
+
+##### Additional software for airgapped deployment
+
+Additionally, if not connected to the public Internet (that is, airgapped); and not connected to a local Yum repository that contains the [additional software](#additional-software), database cluster nodes must also have the following additional software pre-installed:
+
+- libselinux-python (yum package) for python 3.6, [selinux](https://pypi.org/project/selinux/) (PyPI package) for certain higher versions of python (for various Ansible tasks to work under the hood)
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md:42]: Remove these lines

```suggestion
```

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md [hunk 31]
```diff
@@ -0,0 +1,47 @@
...
+
+YBA supports deploying YugabyteDB on a variety of [operating systems](../../../reference/configuration/operating-systems/).
+
+AlmaLinux OS 8 disk images are used by default, but you can specify a custom disk image and OS.
+
+##### Additional software
+
+YBA requires the following additional software to be pre-installed on nodes:
+
+- Python 3.6-3.8
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md:32]:
```suggestion
- Python 3.6-3.8
- Install the python selinux package corresponding to your version of python. For example, using pip, you can install as follows:

`python3 -m pip install selinux`
```

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md [hunk 32]
```diff
@@ -0,0 +1,47 @@
...
+- unzip
+- policycoreutils-python
+
+##### Additional software for airgapped deployment
+
+Additionally, if not connected to the public Internet (that is, airgapped); and not connected to a local Yum repository that contains the [additional software](#additional-software), database cluster nodes must also have the following additional software pre-installed:
+
+- libselinux-python (yum package) for python 3.6, [selinux](https://pypi.org/project/selinux/) (PyPI package) for certain higher versions of python (for various Ansible tasks to work under the hood)
+- libcgroup and libcgrouptools (for planned future use of the cgconfig service, for cgroups; can be omitted for YBA versions earlier than v2024.1)
+- rsync, openssl (all linux)
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md:44]: - semanage-utils (for Debian based systems)
